src/averageAges.js

```diff
@@ -14,12 +14,16 @@
  *
  * @return {number}
  */
+function avaregeAge(peoples) {
+  return peoples.reduce((sum, person) =>
```

Comment:
Plural form - 'people'
```suggestion
  return people.reduce((sum, person) =>
```